backend/jottacloud/jottacloud.go

```diff
@@ -0,0 +1,641 @@
+package jottacloud
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ncw/rclone/backend/jottacloud/api"
+	"github.com/ncw/rclone/fs"
+	"github.com/ncw/rclone/fs/config"
+	"github.com/ncw/rclone/fs/config/configmap"
+	"github.com/ncw/rclone/fs/config/configstruct"
+	"github.com/ncw/rclone/fs/config/obscure"
+	"github.com/ncw/rclone/fs/fserrors"
+	"github.com/ncw/rclone/fs/fshttp"
+	"github.com/ncw/rclone/fs/hash"
+	"github.com/ncw/rclone/lib/pacer"
+	"github.com/ncw/rclone/lib/rest"
+	"github.com/pkg/errors"
+)
+
+// Globals
+const (
+	minSleep          = 10 * time.Millisecond
+	maxSleep          = 2 * time.Second
+	decayConstant     = 2       // bigger for slower decay, exponential
+	defaultDevice     = "Jotta" // TODO ? : Make this a config Option?
+	defaultMountpoint = "Sync"  // TODO ? : Make this a config Option?
+	rootURL           = "https://www.jottacloud.com/jfs/"
+	//newApiRootUrl   = "https://api.jottacloud.com"
+	//newUploadUrl	  = "https://up-no-001.jottacloud.com"
```

Comment:
When I reversed the corresponding upload url for the JFS api (https://www.jottacloud.com/jfs/) one needed to use https://up.jottacloud.com/ to upload (ref https://github.com/havardgulldahl/jottalib/blob/master/src/jottalib/JFS.py#L1128). 

However I can't seem to find it here, did this change later on?

Reply:
both seem to work fine. I'm pretty sure I've seen both being used but I can change it.

Reply:
No need to change it if it works I guess? Was just curious 😃